### ISSUE TITLE
Delete and put methods take longer to process

### ISSUE THREAD

**Issue by dasheck0** [JBlaak/Express-Torch#15]
Hi,

I wrote an api and decided to wire the interface with torch. Torch works as intended and it's nice to describe your API that way. However I found that `delete` and `put` methods take significantly  longer to process. Here is some code to get you started:

```
Torch(app, router => {
        router.group({prefix: 'api/v1'}, router => {

            router.get('/content', contentController.index);
            router.post('/content', {
                middleware: [authenticate, multipartMiddleware],
                controller: contentController.create
            });
            router.put('/content/:id', {
                middleware: [authenticate, multipartMiddleware],
                controller: contentController.update
            });
            router.delete('/content/:id', {
                middleware: [authenticate],
                controller: contentController.delete
            });
        });
    });
```

Getting and posting the content takes about 100ms (measured with postman). However putting and deleting the content takes more than 3 seconds. Removing the middlewares does not affect the response time. When I set the method from delete to get it completes (successfully) in under 100 ms. Also I only returned an empty body in my contentController delete method. Even this takes at least 3 seconds to complete.

Best regards
dasheck

**Comment by JBlaak** [comment]
Can't imagine Torch being the culprit here, there is no difference in handling of, for example, a `GET` request vs. a `PUT` request inside of Torch. Also see: https://github.com/JBlaak/Express-Torch/blob/master/src/connect/transfer.ts

**Comment by JBlaak** [comment]
Now I'm thinking about it, but a `GET` request on a local machine that completes in 100ms is still quite slow. How is your development setup attached to to your node instance? Does it run through nginx or something similar?

**Comment by dasheck0** [comment]
Thanks for the feedback. I looked at your code and I too think that Torch is not the bottleneck here. I didn't have time to do so but I will rewrite the code to the basic app.router approach and see what the results are. 

There is no nginx or such involved in this. The node process is managed by pm2. But I don't think that this is the cause of the problem. I will present the results after the refactoring and then we can see if there is anything to do from there.

**Comment by dasheck0** [comment]
Hey. I just (partially) refactored my code. This is what I did. I removed the delete content route definition from torch. Then I created a new file and used the traditional express router approach. The code looks like the following:

```
var express = require('express');
var router = express.Router();

router.delete('/:id', function(req, res) {
  // the same code
});

module.exports = router;
```

Response time: 47 ms. By the way the earlier mentioned 100ms were just an educated guess. I didn't know the exact values anymore but it were around (or probably less than) 100ms. So the current situation is that if I used Torch I have enormous delays when using put and delete, whether it's due to Torch or not. Maybe I am using it incorrectly (although I used exactly the way it is described in the readme).
